### ISSUE TITLE
Object Detection support from torchvision

### ISSUE THREAD

**Issue by petrhrobar** [muellerzr/fastai_minima#1]
Hey,

Thanks for this amazing wrapper I was trying to implement this myself only to find out this package. I really makes the training loop a pleasure to work with!

I was wondering if there is any support for torchvision model as well - e.g. ```FASTERRCNN_RESNET50_FPN_V2``` model.

Example of the code:

```python
# from torch import optim
from fastai_minima.optimizer import OptimWrapper
from fastai_minima.learner import Learner, DataLoaders
from fastai_minima.callback.core import GatherPredsCallback
train_loader = DataLoader(
    train_dataset,
    batch_size=BatchSIZE,
    shuffle=True,
    num_workers=6,
    collate_fn=collate_fn,
)

test_loader = DataLoader(
    test_dataset,
    batch_size=BatchSIZE,
    shuffle=True,
    num_workers=6,
    collate_fn=collate_fn,
)

dls = DataLoaders(train_loader, test_loader)

model, params, optimizer = get_fast_cnn_model(
    lr=learning_rate,
    n_classes=N_Classes,
    device=device,
    train_params=train_params,
    optimizer_type=optimizer_type,
)

learn = Learner(dls, model)
```

I have my data loader prep the issue I do not have a way around is the loss function in the object detection model itself. It lives already inside the class and thus fastai is having issue inferring the loss.

Honestly, I was not able to provide the losses by hand either. 

Could you please provide any suggestions regarding this?

Thanks again for creating this framework
